modules/layers/src/path-layer/path-layer.ts

```diff
@@ -277,24 +277,24 @@ export default class PathLayer<DataT = any, ExtraPropsT extends {} = {}> extends
   getPickingInfo(params: GetPickingInfoParams): PickingInfo {
     const info = super.getPickingInfo(params);
     const {index} = info;
-    const {data} = this.props;
+    const data = this.props.data as any[];
```

Comment:
Is it really `any`?

Reply:
`any` is good enough for this context. `PickingInfo.object` is expected to be `undefined` if `data` does not resolve to a plain array.